### ISSUE TITLE
libsass doesn't preserve new lines in nested style (Ruby Sass does)

### ISSUE THREAD

**Issue by mzgol** [sass/libsass#552]
The following SCSS:
```scss
a,
div {
    top: 0;
}
```
is generated by Sass 3.4.5 to the following CSS (when output style is set to nested):
```css
a,
div {
  top: 0; }

/*# sourceMappingURL=styles.css.map */
```

Meanwhile, libsass 3.0 converts it to:
```css
/* line 2, /Users/mgol/Documents/projects/bn/cbn/repo/polona-gui/app/styles.scss */
a, div {
  top: 0; }

/*# sourceMappingURL=styles.css.map */
```

Note that `a` and `div` are in the same line in the libsass version.

Another issue with this example: #551

**Comment by mgreter** [comment]
The examples here are 100% passing with my latest refactoring! Although there are some other cases where output is not matching exactly and I had to add the "clean" flag to the new tests suite.

**Comment by am11** [comment]
(I read about the reasoning somewhere libsass issue tracker but forgot what was the consensus). With nested style, libsass converts this:

```scss
a{b:c}
```

to:

```css
a {
  b: c; }
```

while ruby-sass generates:

```css
a {
  b: c;
}
```

(with closing parenthesis at new line)

Should this be fixed to conform with the output of ruby-sass?

**Comment by mgreter** [comment]
Actually with the latest ruby sass this compiles to 

```
a {
  b: c; }
```

Your example looks like you're compiling it in expanded mode!?

**Comment by am11** [comment]
Oh my bad. You are right! I was playing with http://sassmeister.com/, forgot to change to nested. :)

Are there plans to add expanded style in LibSass?

**Comment by mgreter** [comment]
Already done in  #910

**Comment by am11** [comment]
I am an idiot. Sorry and thank you for addressing it! :)

**Comment by mgreter** [comment]
This should be solved by #910!
